feg/protos/s8_proxy.proto

```diff
@@ -176,10 +176,11 @@ message DeleteSessionResponsePgw{
 }
 
 message CreateBearerRequestPgw {
-    uint32 linked_bearer_id = 1;
-    ProtocolConfigurationOptions protocol_configuration_options = 2;
-    BearerContext bearer_context = 3;
-    bytes indication_flag = 4;
+    uint32 c_agw_teid = 1;
```

Comment:
Why add it as field 1? Shouldn't we just add as field 5? Or is this proto not used anywhere in prod.

Reply:
This is not being used yet 
Otherwise i would add it as 5